src/components/hooks/HookDetails.tsx

```diff
@@ -0,0 +1,185 @@
+import React from 'react';
+
+import { createFragmentContainer } from 'react-relay';
+import { createStyles, WithStyles, withStyles } from '@material-ui/core/styles';
+import { RouteComponentProps, withRouter } from 'react-router-dom';
+import { graphql } from 'babel-plugin-relay/macro';
+import { HookDetails_hook } from './__generated__/HookDetails_hook.graphql';
+import { Helmet as Head } from 'react-helmet';
+import { Card, CardContent } from '@material-ui/core';
+import RepositoryNameChip from '../chips/RepositoryNameChip';
+import BuildBranchNameChip from '../chips/BuildBranchNameChip';
+import BuildChangeChip from '../chips/BuildChangeChip';
+import TaskNameChip from '../chips/TaskNameChip';
+import HookCreatedChip from '../chips/HookCreatedChip';
+import Typography from '@material-ui/core/Typography';
+import Logs from '../logs/Logs';
+import HookStatusChip from '../chips/HookStatusChip';
+import CirrusFavicon from '../common/CirrusFavicon';
+import classNames from 'classnames';
+import { useNotificationColor } from '../../utils/colors';
+
+const styles = theme =>
+  createStyles({
+    gap: {
+      paddingTop: theme.spacing(2),
+    },
+    chip: {
+      marginTop: theme.spacing(0.5),
+      marginBottom: theme.spacing(0.5),
+      marginRight: theme.spacing(0.5),
+    },
+    wrapper: {
+      padding: 0,
+      display: 'flex',
+      flexWrap: 'wrap',
+    },
+    io: {
+      padding: theme.spacing(1),
+      background: '#212121',
+      color: '#FFFFFF',
+      fontSize: '12px',
```

Comment:
`theme.typography.fontSize`?

Reply:
See https://github.com/cirruslabs/cirrus-ci-web/pull/355/commits/1b2882221aa371a7a19d805cec26223cce072114.

Reply:
Unfortunately I've had to revert this in https://github.com/cirruslabs/cirrus-ci-web/pull/355/commits/15f0acd43553c066248160e64d5d2ab35ec78c78 because the `theme.typography.fontSize` is slightly larger than the one used in `<Logs>`.

Reply:
Resolved in https://github.com/cirruslabs/cirrus-ci-web/pull/355/commits/95de7ae26f893eb7e0815eefe2987f1a1803a79a.